RoddyCore/test/src/de/dkfz/roddy/config/ConfigurationValueSpec.groovy

```diff
@@ -24,4 +44,32 @@ class ConfigurationValueSpec extends Specification {
         new ConfigurationValue("b", "abc", "string") | null        | evString
         new ConfigurationValue("c", 1)               | null        | evInt
     }
+
+    def "test variable replacement for toFile()"() {
+        when:
+        ExecutionContext context = contextResource.createSimpleContext(ConfigurationValueSpec.class)
+        def values = context.configurationValues
+        values << new ConfigurationValue(context.configuration, "b", 'abc')
+        values << new ConfigurationValue(context.configuration, "c", 'def')
+        values << new ConfigurationValue(context.configuration, "a", '/tmp/${projectName}/${b}/something${c}fjfj/${dataSet}')
+        values << new ConfigurationValue(context.configuration, "complex", '/$USERHOME/$USERNAME/$USERGROUP')
```

Comment:
Should that not also contain exactly the part that was changed? I.e. such that values are not separated by '/'

```groovy
'/bli$USERHOME/bla$USERNAME/blub$USERGROUP'
```

and 

```groovy
'/${USERHOME}bli/${USERNAME}bla/${USERGROUP}blub'
```

Reply:
I think so, really. The whole point of the change is to make exactly this more generic. So a test would be appropriate.

Reply:
The separator does not matter anymore, as the values are not getting split by the resolver. Also, these cases are already covered with value a.